tox.ini

```diff
@@ -51,4 +52,4 @@ commands =
     npm config set spin false
     npm install
     npm run lint
-    npm run package
+    npm run sign
```

Comment:
Actually, maybe I'll just go ahead and do that as I merge, since it's a one line change and this thing needs to get wrapped up!